client/jetpack-cloud/sections/partner-portal/issue-license-v2/review-licenses/index.tsx

```diff
@@ -10,13 +17,25 @@ interface Props {
 export default function ReviewLicenses( { onClose, selectedLicenses }: Props ) {
 	const translate = useTranslate();
 
+	const { sidebarRef, mainRef, initMobileSidebar } = useMobileSidebar();
+
 	return (
-		<DashboardModal
-			title={ translate( 'Review license selection' ) }
-			subtitle={ translate( 'You’re about to issue the following licenses:' ) }
-			onClose={ onClose }
-		>
-			{ selectedLicenses.length }
-		</DashboardModal>
+		<JetpackLightbox isOpen={ true } onClose={ onClose } onAfterOpen={ initMobileSidebar }>
+			<JetpackLightboxMain ref={ mainRef }>
+				<div className="review-licenses__header">
+					<div className="review-licenses__title">{ translate( 'Review license selection' ) }</div>
+					<div className="review-licenses__subtitle">
+						{ translate( 'You’re about to issue the following licenses:' ) }
+					</div>
+					<div className="review-licenses__selected-licenses">
+						{ selectedLicenses.map( ( license ) => (
```

Comment:
Good suggestion, @jkguidaven! I have updated the PR to group licenses by type & sort them by quantity.

> Additionally, I am wondering if it would be possible to combine the product count so that each product type only occupies one space. 🤔 wdyt @jeffgolenski?

I'm not sure how complicated it will be to decide how to combine the bundles. I think we can discuss this & address this in a different PR if needed. WDYT?

Reply:
> I'm not sure how complicated it will be to decide how to combine the bundles. I think we can discuss this & address this in a different PR if needed. WDYT?

sounds good to me 👍

Reply:
@jkguidaven @yashwin Apologies, I was AFK for a holiday. I realize this has been merged and we'll pick up the convo in a different PR, but when you say: "view the products of the same type in the same group" and "and product type only occupies on space" I'm unsure what you mean. Can you elaborate?

Reply:
Hi @jeffgolenski,

I apologize for not being clear earlier. In the screenshot example, we can see that there are two licenses for Akismet Anti-spam - one with 1 license and the other with 10 licenses.  I was thinking To simplify this, we can combine them into one line item and show Akismet Anti-spam with a total of 11 licenses.

<img width="603" alt="Screen Shot 2023-11-23 at 4 50 07 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/3afa878b-b847-4f43-81a6-a09899a36eb0">

I guess one potential concern is that it could be challenging for users to determine which bundles they have selected and the corresponding discounts they are receiving. But let me know what you think.

Reply:
@jkguidaven Ah good find my friend. For now, let's **keep them separate** since this is an itemized list. Hopefully users remember that they selected a bundle and a single, but I wouldn't rely on them if they're purchasing a lot. In any case, I suspect it may be an edge case to see this, but It's still too soon to know!